aws-efs-volume/module_test.go

```diff
@@ -0,0 +1,35 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/chanzuckerberg/cztack/testutil"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestPrivateBucket(t *testing.T) {
+
+	project := testutil.UniqueId()
+	env := testutil.UniqueId()
+	service := testutil.UniqueId()
+	owner := testutil.UniqueId()
+	vpcId := testutil.UniqueId()
```

Comment:
You are going to need to use an id for a real VPC here. There is one set as a variable in the Makefile https://github.com/chanzuckerberg/cztack/blob/master/Makefile#L5